mkdocs-base.yml

```diff
@@ -164,13 +164,13 @@ nav:
           
     - Quickstart guides:
           - Overview: quickstart.md
-          - Quick install:
-            - "Install with Helm": helm.md
-            - "Install with kubectl": kubectl.md
-          - Connect to Percona Server for MongoDB: connect.md
-          - Insert data: data-insert.md
-          - Make a backup: backup-tutorial.md
-          - Monitor the database with PMM: monitoring-tutorial.md
+          - 1. Quick install:
+            - "with Helm": helm.md
+            - "With kubectl": kubectl.md
```

Comment:
the line above starts with lower "w". So depending on capitalization either make this one lower case or capita;ize the "With Helm" title

Reply:
thanks!!!